mesh/src/mutable/Cylindrical2dMesh.cpp

```diff
@@ -160,10 +163,10 @@ void Cylindrical2dMesh::CreateHaloNodes()
     mTopHaloNodes.clear();
     mBottomHaloNodes.clear();
 
-    unsigned num_halo_nodes = (unsigned)(floor(mWidth*2.0));
+    unsigned num_halo_nodes = (unsigned)(floor(mWidth*mHaloScalingFactor));
```

Comment:
Needs to have a formatter run over this: e.g. there should be a single space either side of binary operators such as * and /

The style is contained in the .clang-format file in the repo: all main IDEs will auto-format based on that style file.

Reply:
Fixed this and others I found.